python/dgl/distributed/partition.py

```diff
@@ -187,15 +287,39 @@ def partition_graph(g, graph_name, num_parts, out_path, num_hops=1, part_method=
     If node IDs and edge IDs are not shuffled to ensure that all nodes/edges in a partition
     fall into a contiguous ID range, DGL needs to store node/edge mappings (from
     node/edge IDs to partition IDs) in separate files (node_map.npy and edge_map.npy).
-    The node/edge mappings are stored in numpy files.
+    The node/edge mappings are stored in numpy files. **Note**: this format is
```

Comment:
Use note with the following format:

```
.. note::

   <main-text>
```

Also it's probably better to use `.. warning::` since the interface will be deprecated.